contracts/lib/Errors.sol

```diff
@@ -223,6 +223,8 @@ library Errors {
     error RoyaltyModule__ZeroLicensingModule();
     error RoyaltyModule__CanOnlyMintSelectedPolicy();
     error RoyaltyModule__NoParentsOnLinking();
+    error RoyaltyModule__ZeroDisputeModule();
+    error RoyaltyModule__IpIsTagged();
```

Comment:
Naming nit, `IpIsTagged` vs. `IpIsDisputed` for the tagged error message?

Reply:
There can be a dispute without tagging. We only block tagged assets and not assets in dispute - IpIsDispute could be confusing as it could sound as if a dispute is ongoing in which case it will not revert

Reply:
I agree with the potential confusion. Just trying to make sure from a new dev's perspective: "IpIsTagged" requires knowing what "Tagged" means, which we will have clearly convey in docs.